x-pack/plugins/cases/public/components/header_page/editable_title.tsx

```diff
@@ -118,7 +120,7 @@ const EditableTitleComponent: React.FC<EditableTitleProps> = ({
   ) : (
     <EuiFlexGroup alignItems="center" gutterSize="none" responsive={false}>
       <EuiFlexItem grow={false}>
-        <Title title={title} />
+        <Title title={title} releasePhase={releasePhase} />
```

Comment:
Is it possible to move the release badge to the right of the edit icon? I feel the edit button should live next to the title text as that is what you are editing.